MycroftAI/mycroft-core
# Add info when skill fails to load

## Description
When skill is not loaded and error is raised skill will be loaded to provide any traceback to the test.

## How to test
Edit a skill `__init__` method adding a  `raise Exception` line, run discovertest and check that in the info about the failed tests the traceback is shown.

## Contributor license agreement signed?
CLA [Yes]
